spec/spec_helper.rb

```diff
@@ -1,9 +1,10 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
 
-$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```

Comment:
Doesn't removing this line cause the gem installed version of circuitry to be tested instead of the source version?